### ISSUE TITLE
Missing notifications

### ISSUE THREAD

**Issue by peterweissdk** [Thomas101/wmail#546]
* Which version of WMail are you using? [e.g. 1.3.1]
V. 2.0.9 Prerelease

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
Arch Linux, Gnome 3.22, Wayland window manager.

I dont get any desktop or tray notifications when i resive a new mail!
I have turned mail notifications on in preferences!

Still..., cool app. Finaly an app for my Google Apps.

 peterweissDK

**Comment by Thomas101** [comment]
Hi do you get an unread count over your account icon on the left hand side?

**Comment by peterweissdk** [comment]
Yes I do.
The unread counter on the left side works perfectly.

 Peter

Den 1. feb. 2017 1.52 PM skrev "Thomas Beverley" <notifications@github.com>:

**Comment by Thomas101** [comment]
Thanks! Do you know if you have libnotify installed?

**Comment by peterweissdk** [comment]
I can't really check that right now, because I'm on a holiday, but I will
look in to it, when I get home on sunday.
Thanks for your support.

 Peter

Den 1. feb. 2017 7.57 PM skrev "Thomas Beverley" <notifications@github.com>:

**Comment by peterweissdk** [comment]
Hey Thomas

I have libnotify installed!

$ pacman -Ss libnotify
extra/libnotify 0.7.7-1 [installeret]
    Library for sending desktop notifications
extra/python2-notify 0.1.1-13
    Python bindings for libnotify
community/ibamtray 0.02-9
    system tray ibam frontend based off vubat with libnotify support
community/pidgin-libnotify 0.14-10
    Pidgin plugin that enables popups when someone logs in or messages you

I have sendt you an screenshort of the WMail gui combined with my systray,
where you can see the counter reads 1 unread mail, while I dont get a
notifycation in the systray.

 Peter

 Peter

2017-02-02 7:45 GMT+01:00 Peter Weiss <peterweissdk@gmail.com>:

**Comment by Thomas101** [comment]
Thanks for checking that. If you got to `View > Developer Tools` then click on the `console` tab and paste...

```js
new Notification('test')
```

do you get a notification coming up with the title test?

**Comment by peterweissdk** [comment]
Yes, I get a nofitication from the top panel, but it has no effect on the
systray!



​

2017-02-07 10:18 GMT+01:00 Thomas Beverley <notifications@github.com>:

**Comment by Thomas101** [comment]
Ah okay, I think we have our wires crossed. Do you mean in the app-icon area in the launcher like in the screenshot?

<img width="65" alt="screen shot 2017-02-08 at 17 14 32" src="https://cloud.githubusercontent.com/assets/103586/22748480/1fdab0b6-ee22-11e6-83c0-f9867bdb381b.png">

**Comment by peterweissdk** [comment]
No, I don't have the wmail icon in my launcher, like you show in your
screenshot. But I have a wmail icon in my system tray, which is specific to
the "classic Gnome 3 desktop" and not the Unity desktop.
I thought that the counter would work in system tray, but I was obviously
wrong.
I try to add the wmail icon to my app launcher, and let you know if it
works!

 Peter

Den 8. feb. 2017 6.15 PM skrev "Thomas Beverley" <notifications@github.com>:

**Comment by funilrys** [comment]
Interesting case @Thomas101 :) Do I have to specify libnotify in dependencies?

**Comment by l33tissw00t** [comment]
Not sure if I should be making a new ticket here.. sorry if I should have.

Running Windows 7 64 bit.  Do not receive any notifications of new mail(s) received.  Tray icon does show number of unread emails. Right-clicking on tray icon does bring up list of accounts, with correct number o unread emails per account.

Thanks!
